src/components/LineLayer.tsx

```diff
@@ -69,8 +71,20 @@ export const LineLayer: FunctionComponent<Props> = ({
       lineData: simplifiedLineData,
       interpolation,
       lineWidth,
+      shadeBelow,
+      shadeBelowOpacity,
+      y0: yScale(0),
```

Comment:
Should this be `y0:yScale(yDomain[0])` instead? Or do we always want to shade until zero? Also `y0` is a bit hard to understand what it's for, could it be `shadeAboveY`maybe?

Reply:
Ah yep totally, nice catch. I will rename as well.